internal/provider/resource_consul_cluster.go

```diff
@@ -129,6 +129,13 @@ func resourceConsulCluster() *schema.Resource {
 					return strings.ToLower(old) == strings.ToLower(new)
 				},
 			},
+			"auto_hvn_to_hvn_peering": {
+				Description: "Enables automatic HVN to HVN peering when creating a secondary cluster in a federation.",
+				Type:        schema.TypeBool,
+				Default:     false,
```

Comment:
Oh I realize what you meant by your other question. You can remove `Default` because `Computed` is set. `Default` is the provider picking a default value, `Computed` is the API picking a default value. Generally it's best to just let the API set all defaults, so you don't need to worry about keeping the terraform defaults in sync with the backend.
```suggestion

```